internal/app/blockd/main.go

```diff
@@ -9,9 +9,7 @@ import (
 	"log"
```

Comment:
The changes in this file are to remove the old way of generating the PKI. Each application used to generate a certificate. Instead, the `init` now does all PKI bootstrapping and the services use what is handed to them.